### ISSUE TITLE
Accessibility issues on page.html

### ISSUE THREAD

**Issue by snidersd** [jupyter/notebook#3217]
Also see: [jupyter/notebook#1801](https://github.com/jupyter/notebook/issues/1801)
After running the Dynamic Assessment Plugin for Chrome and the Digital Content Checker on /localhost/tree the following issues were reported missing and can be corrected as indicated:

1. Indicate the `<main>` region of the page.
2. Add a `<lang>` attribute to the tag.

**Comment by takluyver** [comment]
Thanks for identifying all these things! Unlike some of the other issues, though, it's not clear to me what these mean. Does 'region' refer to geography or part of the screen? How do we indicate it? And which tag needs what attribute added?

**Comment by takluyver** [comment]
I just spotted the HTML tags in the original post. I've edited it to make them visible. You need to put backticks around them, or they vanish:

```
`<main>`
```

I'm still not sure what the `<main>` region is, but that's probably google-able now.

**Comment by snidersd** [comment]
Adding the main element to a web page allows a screenreader user to skip to the main content. Here is a link to additional information on the use of main. https://www.w3.org/TR/html-main-element/

**Comment by jtpio** [comment]
Closing as there are now different issues tracking accessibility improvements for Jupyter Notebook 7, for example https://github.com/jupyter/notebook/issues/6800

Thanks!
